libcontainer/cgroups/systemd/v2.go

```diff
@@ -168,7 +173,18 @@ func (m *unifiedManager) Destroy() error {
 	if err != nil {
 		return err
 	}
-	dbusConnection.StopUnit(getUnitName(m.cgroups), "replace", nil)
+	unitName := getUnitName(m.cgroups)
+	statusChan := make(chan string, 1)
+	if _, err := dbusConnection.StopUnit(unitName, "replace", statusChan); err == nil {
+		select {
+		case s := <-statusChan:
+			if DbusJobCompletionResultType(s) != DbusJobDone {
+				logrus.Warnf("error stopping unit `%s`: got `%s`. Continuing...", unitName, s)
```

Comment:
I think `s/stopping/removing/` is better.

Reply:
done!